### DISCUSSION TITLE
how to fnd ganche wallets

### DISCUSSION THREAD

**Discussion by Cpadilla229** [smartcontractkit/full-blockchain-solidity-course-py#1437] [Q&A]
hI have ganche wallets all without funds, how can I fund them? 

ValueError: {'message': "sender doesn't have enough funds to send tx. The upfront cost is: 7322620000000000 and the sender's account only has: 0", 'code': -32000, 'data': {'stack': "Error: sender doesn't have enough funds to send tx. The upfront cost is: 7322620000000000 and the sender's account only has: 0\n    at VM.<anonymous> (/Applications/Ganache.app/Contents/Resources/static/node/node_modules/ganache-core/node_modules/ethereumjs-vm/lib/runTx.ts:114:11)\n    at step (/Applications/Ganache.app/Contents/Resources/static/node/node_modules/ganache-core/node_modules/ethereumjs-vm/dist/runTx.js:33:23)\n    at Object.next (/Applications/Ganache.app/Contents/Resources/static/node/node_modules/ganache-core/node_modules/ethereumjs-vm/dist/runTx.js:14:53)\n    at fulfilled (/Applications/Ganache.app/Contents/Resources/static/node/node_modules/ganache-core/node_modules/ethereumjs-vm/dist/runTx.js:5:58)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)", 'name': 'Error'}}

**Comment by cromewar** [comment]
that's weird as ganache comes with pre funded accounts, so reinstalling ganache should work.
